tests/mobly/logger_test.py

```diff
@@ -23,8 +24,12 @@ class LoggerTest(unittest.TestCase):
     """Verifies code in mobly.logger module.
     """
     def test_epoch_to_log_line_timestamp(self):
-        # actual_stamp = logger.epoch_to_log_line_timestamp(1469134262116)
-        # self.assertEqual("07-21 13:51:02.116", actual_stamp)
+        correct_stamp = (
+            datetime.fromtimestamp(1469134262.1161)
```

Comment:
Just saw the PR comment. It definitely needs a code comment at least. 

epoch_to_log_line_timestamp has the same issue with py3.4, and handles this by removing the milliseconds, rounding them manually, and appending them onto the end of the formatted string. You could do the same thing, but as this test becomes more flexible, it is starting to duplicate the logic of the method under test. Since it's a formatting method, I do think comparing to a hardcoded string in the right format is ultimately the right thing to do here.

fromtimestamp takes an optional timezone object. Maybe the right thing to do is to expose tz in epoch_to_log_line_timestamp (default None) and supply UTC in the unit test. Then we can do the entire test explicitly in UTC and compare to a hardcoded string.

That way we get timezone independence, but don't have a 'test' that just duplicates the formatting logic of the method under test.

What do you think?